bson/src/main/org/bson/BsonDocument.java

```diff
@@ -782,6 +782,17 @@ public String getFirstKey() {
         return keySet().iterator().next();
     }
 
+    /**
+     * Gets the first value in the document
+     *
+     * @return the first value in the document
+     * @throws java.util.NoSuchElementException if the document is empty
+     * @since 3.7
+     */
+    public BsonValue getFirstValue() {
```

Comment:
Added for efficiency of getting the first value to log.  Not important here, but it is for subclasses that override this.